lib/beaker-puppet/helpers/rake_helpers.rb

```diff
@@ -0,0 +1,21 @@
+module Beaker
+  module DSL
+    module Helpers
+      # Methods that help you interact with rake during ci setup
+      module RakeHelpers
+        class << self
+          def load_tasks(beaker_root = '/Users/melissa/beaker-puppet')
```

Comment:
can we make it a required parameter?

Reply:
oh shoot, I meant to fix this, but I totally forgot. Thank you! I don't think we need to make it required, since we should be able to introspect where beaker-puppet is being run. I do that for the setup tests